src/src/PluginStructs/P093_data_struct.cpp

```diff
@@ -359,6 +370,24 @@ void P093_data_struct::applySettings() {
     packet[7] |= 0x01;
   }
 
+  if (_writeStatus.isDirty(RemoteTemperature)) {
+    packet[5] = 0x07;
+    if(setting > 0) {
+      packet[6] = 0x01;
```

Comment:
if (_writeStatus.isDirty(RemoteTemperature)) {
    packet[5] |= 0x07;
    if(_wantedSettings.remoteTemperature > 0) {
      if (_tempMode) {
        packet[8] = static_cast<uint8_t>(_wantedSettings.remoteTemperature * 2.0f + 128.0f);
      } else {
        packet[7] = _wantedSettings.remoteTemperature;
      }
    }
    else {
      packet[6] = 0x00;
      packet[8] = 0x80; //MHK1 send 80, even though it could be 00, since ControlByte is 00
    } 
  }

Reply:
```c++
packet[8] = static_cast<uint8_t>(_wantedSettings.remoteTemperature * 2.0f + 128.0f);
```
That's never going to fit in an `uint8_t`, unless the remote temperature is -63 ... 0